### PR TITLE
[CI-26][CI-27] Basic file upload

### PR DESCRIPTION
**Pull request by rcybulski1122012**
I created an endpoint instead of a script to create a quiz, but I think the interface generated based on OpenAPI is more convenient (code smells must be provided as a json).

### CODE REVIEW

File: backend/.env.template [hunk 1]
```diff
@@ -6,3 +6,6 @@ POSTGRES_DB="fastdb"
 PGADMIN_EMAIL="fastaexample@gmail.com"
 PGADMIN_PASSWORD=123
 POSTGRES_URL=postgresql+asyncpg://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+BLOB_STORAGE_CONNECTION_STRING=
+BLOB_STORAGE_CONNECTION_STRING="ceramicyio"
+BLOB_ACCOUNT_NAME="ceramicyio"
```
Comment (rcybulski1122012) [inline backend/.env.template:11]: Let me know if you want to get secrets 😃

File: backend/app/routers/file.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter, UploadFile
+
+from app.services.file import upload_single_file
+
+router = APIRouter(tags=["File"], prefix="/file")
+
+
+@router.post("/")
+async def upload_file(file: UploadFile) -> list[str]:
```
Comment (rcybulski1122012) [inline backend/app/routers/file.py:9]: I've tried to implement in with a single endpoint, but `UploadFile` uses `Form` and it cannot be combined with `Body` (HTTP protocol restriction)

File: backend/app/services/file.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        await blob_client.upload_blob(file_bytes)
+
+    return _get_blob_url(blob_path)
+
+
+def _check_if_text_based_file(file: bytes, mime_type: str) -> None:
+    try:
+        file.decode("utf-8")
+    except UnicodeDecodeError:
+        raise HTTPException(status.HTTP_400_BAD_REQUEST, f"File with mime type {mime_type} is not text-based")
```
Comment (rcybulski1122012) [inline backend/app/services/file.py:29]: I chose to use this simple heuristic not to have to generate/get all the mime-types of text files. Hope it'll work well (already tested on basic file types)

**Review by IwoSzczepaniak — APPROVED** [review]
Ok!

**Review by socz3qqq — APPROVED** [review]
Looks good!

**Review by BARBUR02 — APPROVED** [review]
LGTM

**Pull request merged**